strapi/strapi
# fix: wrong jobs value after removing a task

### What does it do?
In cron service remove method, the array.filter method is replaced with array.splice method so the jobs property would point to the correct value.

### Why is it needed?
In cron service remove method, array.filter method was used which created a shallow copy of a jobsSpec array. This meant that the jobs property pointing to the jobsSpec array was not updated. So after removing a task, jobs returned the old array with the removed task.

### How to test it?
Add a cron task, log the jobs array, remove a cron task, log the jobs array. After the fix, removed task is no longer present in the jobs array.

### Related issue(s)/PR(s)
None
